### ISSUE TITLE
Move tkdnd to Dropp submodule

### ISSUE THREAD

**Issue by rdbende** [tukaan/tukaan#88]
TODO:
- [ ] Move `tkdnd` to `Dropp` submodule
- [ ] Rename `tkdnd` to `Dropp`
- [ ] Improve build system (help needed)
- [ ] Translate Tcl part to Python for easier maintainability

**Comment by rdbende** [comment]
It's the extension that adds Drag and drop capabilities to Tukaan.

**Comment by Moosems** [comment]
Like dragging of files or widget movement?

**Comment by rdbende** [comment]
Yes. Files, text, and basically anything that can be dragged (icons from my dock, HTML elements from the browser, color swatches from Inkscape, ...)

**Comment by Moosems** [comment]
Oh wow

**Comment by Moosems** [comment]
By `Move `tkdnd` to `Dropp` submodule` do you want me to copy/paste the files and then start translating it to python?

**Comment by Moosems** [comment]
I don't see the Dropp submodule anymore

**Comment by rdbende** [comment]
The repo is now public
